ui/components/app/wallet-overview/token-overview.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { generateUseSelectorRouter } from '../../../hooks/gasFeeInput/test-utils';
+import { renderWithProvider } from '../../../../test/jest/rendering';
+import TokenOverview from './token-overview';
+
+jest.mock('react-redux', () => {
+  const actual = jest.requireActual('react-redux');
+
+  return {
+    ...actual,
+    useSelector: jest.fn(),
```

Comment:
I've seen this in a few places so am just curious about this. What is the benefit of mocking `useSelector` rather than each selector function that this component uses? I'm wary of mocking Redux library code.

Reply:
Rather than mocking `useSelector` we should create a mock store that actually fits the test needed. Done here: https://github.com/MetaMask/metamask-extension/pull/13291/commits/4139652022a579b67556f3783bfbea1ddfc4e609